src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -49,8 +65,9 @@ function logCapturedError(capturedError : CapturedError) : void {
 
     console.error(
       `${componentNameMessage} You should fix this error in your code. ${errorBoundaryMessage}\n\n` +
-      `${error.stack}\n\n` +
-      `The error was thrown in the following location: ${componentStack}`
+      `${errorSummary}\n\n` +
+      `The error is located at: ${componentStack}\n\n` +
+      `The error was thrown at: ${formattedCallStack}`
```

Comment:
Edge-case in message content and browser formatting differences are why I initially just logged `error.stack` by itself. The "thrown at" header and stripping the redundant error message (in the case of Chrome) were suggestions from others.

FWIW I think the stack, by itself, is not as intuitive without the header and formatting for certain browsers.